docs/connector-development/config-based/overview.md

```diff
@@ -209,5 +205,5 @@ pagination_strategy:
 The following connectors can serve as example of what production-ready config-based connectors look like
 
 - [Greenhouse](https://github.com/airbytehq/airbyte/tree/master/airbyte-integrations/connectors/source-greenhouse)
-- [Sendgrid](https://github.com/airbytehq/airbyte/tree/master/airbyte-integrations/connectors/source-sendgrid)
-- [Sentry](https://github.com/airbytehq/airbyte/tree/master/airbyte-integrations/connectors/source-sentry)
+- [Sendgrid](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-sendgrid/source_sendgrid/sendgrid.yaml)
```

Comment:
update links to point to the yaml file for sendgrid and sentry because the connector is not using low-code